app/src/main/java/com/example/nuberjam/utils/Constant.kt

```diff
@@ -14,4 +14,8 @@ object Constant {
 
     const val LIBRARY_LINEAR_TYPE = "linear"
     const val LIBRARY_GRID_TYPE = "grid"
+}
+
+enum class LibraryDetailType(val code: Int) {
+    FAVORITE(1), ALBUM(2), PLAYLIST(3)
```

Comment:
Better to use int or string? If we use string it can use the lowercase version value like "favorite"

Reply:
I dont any idea, what int or string do you mean?